neo4j-monitoring/templates/service.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "neo4j-monitoring.fullname" . }}
+  labels:
+{{ include "neo4j-monitoring.labels" . | indent 4 }}
+spec:
```

Comment:
Don't you have to match the service with the endpoint somehow?

Reply:
Service & Endpoints have a match by name.